### PR TITLE
fix: initial code

### PR DESCRIPTION
**Pull request by chandanaannadi**

### CODE REVIEW

**Review by Neelima1415 — CHANGES_REQUESTED** [review]
1. Add default to the nav-bar to come to the responsive mode
2. Set the alignment for header

**Review by DathuKasturi834 — CHANGES_REQUESTED** [review]
set proper allignment to the carousel

File: bootstrap code/index.html [hunk 1]
```diff
@@ -104,6 +107,7 @@ <h3>Water Pollution</h3>
             <a class="right carousel-control" data-slide="next" href="#myCarousel"><span class="icon-next"></span></a>
         </div>
     </div>
+</div>
     <div class="container" id="bigCallout">
```
Comment (swayampakula) [inline bootstrap code/index.html:111]: Remove the click button as it does not do anything

**Review by swayampakula — APPROVED** [review]
Remove the click button and then push back to the branch and merge the code.

**Pull request merged**